### ISSUE TITLE
EmbAJAXSlider only updating value when releasing mouse left-click

### ISSUE THREAD

**Issue by Rhvarrier** [tfry-git/EmbAJAX#23]
EmbAJAXSlider instance is only updating value when releasing mouse left-click. Is is possible to update its value instantaneously when moving the slider (same goes for the joystick)

**Comment by tfry-git** [comment]
Same as #14 (and sorry for still not having an official solution).

Changing onChange to onInput on this line: https://github.com/tfry-git/EmbAJAX/blob/98cfc9b6f182df4e868b2cb21e75b4c89eb1f641/EmbAJAX.cpp#L248 will work as a quick-and-dirty solution.

**Comment by FedericoBusero** [comment]
Hi,

I made an implementation which updates the values of the slider (and the color picker) each x ms when it has been changed. However, I am no sure what is the best way to change the API. Is backward compatibility necessary? My proposal is

EmbAJAXSlider::EmbAJAXSlider(const char* id, int16_t min, int16_t max, int16_t initial, int16_t update_min_interval_ms=0);
EmbAJAXColorPicker::EmbAJAXColorPicker(const char* id, uint8_t r, uint8_t g, uint8_t b, int16_t _update_min_interval_ms=0);

a) When  update_min_interval_ms equals zero (or when it is not specified) the functionality remains the same as today. This way, full reverse compatibility is guaranteed
b) When a time interval is specified, it will update the value with minimum time between 2 updates, except for the last (release) update which comes undelayed
